src/olympia/reviewers/models.py

```diff
@@ -1170,3 +1170,15 @@ class ThemeLock(ModelBase):
 
     class Meta:
         db_table = 'theme_locks'
+
+
+class PrivateWhiteboard(ModelBase):
```

Comment:
This would have been a nice occasion to also move the existing `whiteboard` content from `Addon` to this new model you're adding - in which case the name doesn't match. That would have avoided some extra copy/paste (you could have re-used the existing whiteboard views, setting them both at once)... but out of scope for this, so ignore this comment.